src/main/java/org/geomerty/oop/shapes/Circle.java

```diff
@@ -0,0 +1,29 @@
+package org.geomerty.oop.shapes;
+
+import static java.lang.Math.PI;
+
+public class Circle {
+    private final Point center;
+    private final double radius;
+
+    public Circle(Point center, double radius) {
+        this.center = center;
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public Point getCenter() {
+        return center;
+    }
+
+    public double area() {
```

Comment:
I'm not sure, but I think calling the getRadius() method takes more processing power than just getting the radius straight.